### DISCUSSION TITLE
[Feature Request] Progress Spinner - determinate mode

### DISCUSSION THREAD

**Discussion by tsteuwer-accesso** [primefaces/community#1479] [PrimeNG] [answered]
Currently, the [Progress Bar](https://primeng.org/progressbar) has a determinate mode where you can set the `value` property to a number between `0` and `100` and the bar moves accordingly.

Can we also have this option for the Spinner as well?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, we'll wait for community upvotes for this.
